scripts/lite/shelley-testnet.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Have you run the `shellcheck` program over this?

Reply:
Thanks for the suggestion.

I have applied `shellcheck` recommendations.